### ISSUE TITLE
📚 Course Topic Submission

### ISSUE THREAD

**Issue by mathesond2** [buildspace/buildspace-projects#31]
### Course Topic
* deeper dive into Solidity smart contracts
     *  the web3 course has been GREAT to get an overview of common FE tooling and a bit of smart contract writing...it would be great if we could have a follow-up as a deeper dive into writing smart contracts.
     * ex: breaking down a well-known, existing smart contract by finding it on etherscan and using it to teach holistic concepts beyond syntax. These concepts could further be cemented as part of more complicated user project. As a follow-up, users (or groups of users) could do their own independent research on a smart contract of their choice and create their own smart contract(s) using what they've learned. 

### Have You Taken a Course On This Topic Before
* nope!

### Do You Want To Write This Course?
* don't think I'd be qualified to write it, but would definitely be down to pair up and learn

**Comment by adilanchian** [comment]
Heck ya! Thanks so much for submitting this!!!

Will chat with the team and see what we can do here :)

**Comment by marcelc63** [comment]
Hi wanted to create a course suggestion but this one right here is the same idea as mine. (I voiced mine on #feedback at discord). So commenting here to affirm my support & demand for content like this.

What I wrote on discord:

"Hi @_buildspace Team , I have an idea. Since most smart contracts are inspectable, perhaps buildspace can have a session/series that breaks down popular smart contracts and point out best practices. E.g., deep dive into CryptoPunks' contract and understand how it was written."

Thanks!
